src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,21 @@
-export const mappings = {
+const defaultMappings = {
+  '_default_': {
+    'dynamic': 'strict'
+  },
   config: {
+    dynamic: true,
     properties: {
       buildNum: {
         type: 'keyword'
       }
     }
   },
-  server: {
-    properties: {
-      uuid: {
-        type: 'keyword'
-      }
-    }
-  }
 };
+
+export function assignMappings(mappings) {
+  Object.assign(defaultMappings, mappings);
```

Comment:
Are concerned with, or should we provide protections against, someone overwriting the config property?